configuration/pih/liquibase/sql/create_functions.sql

```diff
@@ -800,7 +800,7 @@ END
 #
 
 -- This function accepts encounter_id, mapping source, mapping code
--- It will find a single, best observation that matches this, and return the value_text
+-- It will find a single, best observation that matches this, and return the concept name
```

Comment:
Just changed this to clarify that is returns the concept_name of the value_coded, not the value_text (since value_text is something different)